controllers/ovndbcluster_controller.go

```diff
@@ -612,6 +622,16 @@ func (r *OVNDBClusterReconciler) reconcileNormal(ctx context.Context, instance *
 
 		// Set DB Address
 		instance.Status.InternalDBAddress = strings.Join(internalDbAddress, ",")
+		if instance.Spec.DBType == ovnv1.SBDBType && instance.Spec.NetworkAttachment != "" {
+			// This config map will populate the sb db address to edpm, can't use the nb
+			// If there's no networkAttachments the configMap is not needed
```

Comment:
why would we not unconditionally create the config map? I don't think anything breaks if we do, and one may want to inspect the configMap even if the SB pod is not attached to an additional network, yet. Not sure why we have to condition it.

Reply:
OK I see it was suggested in another thread. I commented there: https://github.com/openstack-k8s-operators/ovn-operator/pull/352#discussion_r1783177268 but I won't insist on a change here. :)